albumentations/augmentations/functional.py

```diff
@@ -1964,3 +1964,9 @@ def fancy_pca(img, alpha=0.1):
     orig_img = orig_img.astype(np.uint8)
 
     return orig_img
+
+
+@clipped
+def shot_noise(image, shotnoise):
+
+    return shotnoise
```

Comment:
Are you sure that we are need there empty function? I think is better call `clipped` function in `get_params_dependent_on_targets`

Reply:
I thought it is necessary to have a function in "functional" script. I can do as you are suggesting.